Documentation/ApiOverview/DependencyInjection/Index.rst

```diff
@@ -714,6 +714,36 @@ be declared public:
     Call to a member function methodName() on null
 
 
+How to override service arguments?
+----------------------------------
+
+Some services in the TYPO3 core use service arguments, which can be overridden by third-party extensions.  
+For example, the :php:`$rateLimiterFactory` argument in the :php:`PasswordRecoveryController` of the 
```

Comment:
```suggestion
For example, the :php:`$rateLimiterFactory` argument in the :php-short:`\TYPO3\CMS\FrontendLogin\ControllerPasswordRecoveryController` of the 
```

Using the FQN in combination with php-short will only print `ControllerPasswordRecoveryController` but enable users to open the class on GitHub with two clicks